sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/ModelsRepositoryAsyncClient.java

```diff
@@ -62,4 +63,64 @@
     public ModelsRepositoryServiceVersion getServiceVersion() {
         return this.serviceVersion;
     }
+
+    /**
+     * Gets a collection of model definitions.
+     *
```

Comment:
extra line

Reply:
that is auto formatting and standard, there is a new line after description and before parameters.